modules/lvm2/udiskslvm2moduleiface.c

```diff
@@ -154,7 +154,6 @@ lvm_update_from_variant (GPid      pid,
           udisks_linux_volume_group_object_destroy (group);
           g_dbus_object_manager_server_unexport (manager,
                                                  g_dbus_object_get_object_path (G_DBUS_OBJECT (group)));
-          g_object_unref (G_OBJECT (group));
```

Comment:
Sounds good to me as long as no other thread could be accessing the same hash table. Otherwise it may cause the object to be destroyed (freed) while we have this has a reference to it without increasing the ref counter, or am I missing something?